### ISSUE TITLE
No name is defined for All cleared button in Task section in Home screen: A11y_.NET 9 template_Home_Name Role Value.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#26896]
### GitHub Tags:
#A11yTCS; #A11ySev2; #Android; #WCAG4.1.2; #SH_.NET9template_Android_Dec2024; #Talkback; #.NET Core; #Device; #A11yMAS; #Name Role Value; #AINotInscope;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
Android Version: 14
Talkback version: 15.1

### Repro Steps:

1. Turn on Talkback and Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Swipe till "Task section".
4. swipe till list items under task section.
5. Select now 2 list items in Task section by double tapping on it, once selected All cleared button will appear.
6. Now Observe that No name is defined for All cleared button in Task section in Home screen.

### Actual Result:
No name is defined for All cleared button in Task section in Home screen.

### Expected Result:
Name should be defined for All cleared button in Task section in Home screen.

### User Impact:
Users who relay on screen reader wont get proper information of that All cleared button.

### Attachment:
[26896_A11y_.NET 9 template_Home_Name Role Value..zip](https://github.com/user-attachments/files/18285104/26896_A11y_.NET.9.template_Home_Name.Role.Value.zip)

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by keerthiduvvuri** [comment]
Verified in latest Environment, Issue is Fixed. Hence closing the bug.
Added attachment for reference.
#Regressed:03-06-25; #Closed; #A11yFixed;

[26896_Closing Attachment.zip](https://github.com/user-attachments/files/19102263/26896_Closing.Attachment.zip)
